stable/aws-for-fluent-bit/templates/clusterrole.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.serviceAccount.create -}}
+{{- if .Values.rbac.create -}}
```

Comment:
I would actually just remove the `{{ if }}` guard entirely from the `clusterrole.yaml` and `clusterrolebinding.yaml` templates. None of the other charts have this guard on either their `clusterrole[binding].yaml` or combined `rbac.yaml` template files, FWICT.

Reply:
Yes agree. I'll entirely remove them